static/docs/commands-reference/remote:add.md

```diff
@@ -75,31 +75,36 @@ is specified for them.
 
 <details>
 
-### Click for LOCAL example
+### Click for a local remote example
 
-Using a relative path:
+> While the term may seem contradictory, it doesn't have to be.
+The "local" part refers to the machine where the project is stored, so it can be
+any directory accessible to the same system. The "remote" part refers
+specifically to the project/repository itself.
+
+Using an absolute path (recommended):
 
 ```dvc
-    $ dvc remote add myremote ../dir
+    $ dvc remote add myremote /path/to/dir
```

Comment:
let's use something like `/tmp/dvc-example-storage`

so that command is actually executable

Reply:
Went for `/tmp/my-dvc-storage`